<?xml version="1.0"?>
<launch>
  <arg name="rviz" default="true"/>
  <include file="$(find arx_mini_gazebo)/launch/two_mobile_manipulators_gazebo.launch"/>
  
  <node name="dwa" pkg="arx_mini_multi_robot" type="dwa.py" output="screen" >
    <param name="selfname" value="arx2" />
    <param name="target" value="arx1 1.0 0.0" />
  </node>
  
  <include file="$(find arx_mini_navigation)/launch/localization.launch">
    <arg name="robot_name" value="arx1"/>
    <arg name="map" default="gazebo.yaml" />
  </include>
  
  <include file="$(find arx_mini_navigation)/launch/localization.launch">
    <arg name="robot_name" value="arx2"/>
    <arg name="map" default="gazebo.yaml" />
  </include>
  
  
  <node name="map_to_map1" pkg="tf2_ros" type="static_transform_publisher" args="0 0 0 0 0 0 1 map arx1/map"/>
  <node name="map_to_map2" pkg="tf2_ros" type="static_transform_publisher" args="0 0 0 0 0 0 1 map arx2/map"/>
  
  <node if="$(arg rviz)" pkg="rviz" type="rviz" name="rviz" required="true"
        args="-d $(find arx_mini_multi_robot)/rviz/multi.rviz"/>
</launch>
